### PR TITLE
create test new position

### PR DESCRIPTION
**Pull request by at-hanhnguyen2**
create test new position

### CODE REVIEW

File: src/test/java/stepdefs/position/NewPositionDefinitions.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
+package stepdefs.position;
+
+import cucumber.api.java8.En;
```
Comment (at-vinhhuynh) [inline src/test/java/stepdefs/position/NewPositionDefinitions.java]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/akaj-team/selenium-test/blob/914b99008e5373aaa539a437040b6e6718a1804a/src/test/java/stepdefs/position/NewPositionDefinitions.java#L3"></span>Unused import - cucumber.api.PendingException.</td>
    </tr>
  </tbody>
</table>

File: src/test/resources/features/position/NewPosition.feature [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    Then Hide confirmation dialog
+
+  Scenario: Test dialog confirmation when click leave
+    Given I enter correct long name with text is "HHHHHHHH"
+    And I click first position on list position
+    When I click on button position
+    Then Should show dialog exits confirmation
+    When I click button leave
+    Then Should redirect to position page
+
```
Comment (ml-anh-quach) [inline src/test/resources/features/position/NewPosition.feature]: remove this line

File: src/test/java/vn/asiantech/page/position/NewPositionPage.java [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+public class NewPositionPage extends BasePage<NewPositionPage> {
+
+    @FindBy(id = "btn-link-to-position-list")
+    private WebElement btnPosition;
+    @FindBy(xpath = "//input[contains(@formcontrolname,'long_name')]")
+    private WebElement edtLongName;
+    @FindBy(xpath = "//input[contains(@formcontrolname,'short_name')]")
+    private WebElement edtShortName;
+    @FindBy(id = "position-form-wrapper")
+    private WebElement form;
```
Comment (ml-anh-quach) [inline src/test/java/vn/asiantech/page/position/NewPositionPage.java]: define name clearly

File: src/test/java/vn/asiantech/page/position/NewPositionPage.java [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+
+    public final void inputShortName(final String shortNameInput, final int firstSpace, final int lastSpace) {
+        sendKey(edtShortName, shortNameInput, firstSpace, lastSpace);
+    }
+
+    public final void inputCorrectLongName(final String longName) {
+        edtLongName.sendKeys(longName);
+    }
+
+    public final boolean checkMessageError(final String message, final int pos) {
```
Comment (at-hangtran) [inline src/test/java/vn/asiantech/page/position/NewPositionPage.java]: All function that return  `boolean` type should have prefix `is`. Please rename `checkMessageError` to `isMessageErrorCorrect`

File: src/test/java/vn/asiantech/page/position/NewPositionPage.java [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+    private WebElement formMainPositionPage;
+    @FindBy(id = "btn-submit-position")
+    private WebElement btnSubmit;
+    @FindBy(css = ".app-alert.ng-star-inserted")
+    private WebElement alertError;
+    @FindBy(className = "ui-dialog")
+    private WebElement alertConfirmation;
+
+    private WebDriver driver;
+    private static final int POS_LIST_POSITION = 2;
```
Comment (at-hangtran) [inline src/test/java/vn/asiantech/page/position/NewPositionPage.java]: Please place all constants at the top of classes

**Pull request merged**